Crash report 2.9.1: SQLiteException in WordPressDB.<init>

```
java.lang.RuntimeException: Unable to create application org.wordpress.android.WordPress: android.database.sqlite.SQLiteException: duplicate column name: blogId (code 1): , while compiling: alter table accounts add blogId integer;
       at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4635)
       at android.app.ActivityThread.access$1300(ActivityThread.java:162)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1424)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:194)
       at android.app.ActivityThread.main(ActivityThread.java:5371)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:525)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:833)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: android.database.sqlite.SQLiteException: duplicate column name: blogId (code 1): , while compiling: alter table accounts add blogId integer;
       at android.database.sqlite.SQLiteConnection.nativePrepareStatement(SQLiteConnection.java)
       at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:886)
       at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:497)
       at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:588)
       at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
       at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
       at android.database.sqlite.SQLiteDatabase.executeSql(SQLiteDatabase.java:1663)
       at android.database.sqlite.SQLiteDatabase.execSQL(SQLiteDatabase.java:1594)
       at org.wordpress.android.WordPressDB.<init>(WordPressDB.java:167)
       at org.wordpress.android.WordPress.initWpDb(WordPress.java:177)
       at org.wordpress.android.WordPress.onCreate(WordPress.java:113)
       at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1000)
       at android.app.ActivityThread.handleBindApplication(ActivityThread.java:4632)
       at android.app.ActivityThread.access$1300(ActivityThread.java:162)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1424)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:194)
       at android.app.ActivityThread.main(ActivityThread.java:5371)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:525)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:833)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:600)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

--- a/src/org/wordpress/android/WordPress.java
+++ b/src/org/wordpress/android/WordPress.java
@@ -222,8 +222,9 @@ public class WordPress extends Application {
             editor.commit();
             if (wpDB != null) {
                 wpDB.updateLastBlogId(-1);
-                wpDB.deleteDatabase(this);
             }
+            // Force DB deletion
+            WordPressDB.deleteDatabase(this);
             wpDB = new WordPressDB(this);
         }
     }

--- a/src/org/wordpress/android/WordPressDB.java
+++ b/src/org/wordpress/android/WordPressDB.java
@@ -252,7 +252,7 @@ public class WordPressDB {
         return db;
     }
 
-    public void deleteDatabase(Context ctx) {
+    public static void deleteDatabase(Context ctx) {
         ctx.deleteDatabase(DATABASE_NAME);
     }
 

